Fix installation schematics

Installing with the script gives this error:
```
 error TS2305: Module ‘“@spartacus/storefront”’ has no exported member ‘CartPageEventModule’.
```

we need to reflect this change: https://github.com/SAP/spartacus/pull/14502/files#diff-14b359a06c80571d3436cf0a7874dc3f7661b5f6aa60aff6d5d0093085ca00bd

in this file: https://github.com/SAP/spartacus/blob/epic/cart-lib/projects/schematics/src/add-spartacus/spartacus-features.ts

--- a/.github/workflows/install.yml
+++ b/.github/workflows/install.yml
@@ -1,9 +1,9 @@
 on:
   push:
     branches:
-      - 'feature/GH-13889'
       - 'release/3.**'
       - 'release/4.**'
+      - 'epic/**'
       - 'develop'
 name: Installation
 jobs:

--- a/projects/schematics/src/add-spartacus/spartacus-features.ts
+++ b/projects/schematics/src/add-spartacus/spartacus-features.ts
@@ -221,7 +221,6 @@ function configureSpartacusModules(
         `// Page Events,
         NavigationEventModule,`,
         'HomePageEventModule,',
-        'CartPageEventModule,',
         'ProductPageEventModule,',
       ].forEach((content) => {
         addModuleImport(sourceFile, {
@@ -231,7 +230,6 @@ function configureSpartacusModules(
               namedImports: [
                 'NavigationEventModule',
                 'HomePageEventModule',
-                'CartPageEventModule',
                 'ProductPageEventModule',
               ],
             },

